app/index.js

```diff
@@ -52,6 +52,6 @@ app.on("ready", function () {
     creatorFunc()
         .then(creator.check)
         .then(function () {
-            abrApp = new AbrApplication(osxOpenFilePaths);
+            abrApp = new AbrApplication(osxOpenFilePaths, app.getLocale());
```

Comment:
I think you can do `app.getLocale()` directly in `abrApplication` and remove this parameter.